sandbox/settings.py

```diff
@@ -81,7 +81,9 @@ class Base(DRFMixin, MagnifyCoreConfigurationMixin, Configuration):
 
     # Security
     ALLOWED_HOSTS = []
-    SECRET_KEY = values.Value("ThisIsAnExampleKeyForDevPurposeOnly")
+    CSRF_TRUSTED_ORIGINS = values.ListValue([])
+    SECRET_KEY = values.Value(None)
```

Comment:
Are we sure that having `None` as a default won't be a problem if it is not defined ?

Reply:
Yes it will break but that's the purpose? If you don't set it Django also breaks.

Reply:
Yep because Django can't work without a secret key.

I was in doubt on the behavior when it's set to `None` so I checked and it's all good:

![Screenshot from 2022-09-22 10-36-51](https://user-images.githubusercontent.com/1427165/191699761-60b9a41e-d87d-4a31-8e36-349ce8f7d30f.png)